app/utils/types/module_list.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+
+from app.utils.types.groups_type import GroupType
+from app.utils.types.module import Module
+
+
+class ModuleList(Module, Enum):
+    advert = Module(
+        root="advert",
+        default_allowed_groups_ids=[GroupType.student, GroupType.staff],
+    )
+    admin = Module(
+        root="admin",
+        default_allowed_groups_ids=[GroupType.admin],
+    )
```

Comment:
Should admin really be able to disable `admin` module?

Reply:
Right now access to admin module is managed on Titan side but it could be better to add a special case in futur implementation. Right now it may be better to remove it from the module list as is not considered as a drawer module in Titan and does nothing when changed.